docker/scripts/load_data.py

```diff
@@ -217,19 +217,34 @@ def load_error_analysis():
             rg.TextQuestion(name="note", title="Leave a note to describe the issue:", required=False),
         ]
 
-        metadata = [
-            rg.TermsMetadataProperty(name="correctness-langsmith", values=df.correctness_langsmith.unique().tolist()),
-            rg.TermsMetadataProperty(name="model-name", values=df.model_name.unique().tolist()),
-            rg.FloatMetadataProperty(name="temperature", min=df.temperature.min(), max=df.temperature.max()),
-            rg.IntegerMetadataProperty(name="max-tokens", min=df.max_tokens.min(), max=df.max_tokens.max()),
-            rg.FloatMetadataProperty(name="cpu-user", min=df.cpu_time_user.min(), max=df.cpu_time_user.max()),
-            rg.FloatMetadataProperty(name="cpu-system", min=df.cpu_time_system.min(), max=df.cpu_time_system.max()),
-            rg.TermsMetadataProperty(name="library-version", values=df.library_version.unique().tolist()),
-        ]
+        dataset_name = "error-analysis-with-feedback-alpha"
```

Comment:
I think `-alpha` can now be safely removed

```suggestion
        dataset_name = "error-analysis-with-feedback"
```